docs/webpack.config.js

```diff
@@ -1,10 +1,10 @@
 const path = require('path');
 
 module.exports = {
+  mode: "production",
   entry: "./bootstrap.js",
   output: {
-    path: path.resolve(__dirname, "static"),
+    path: path.resolve(__dirname, "static", "playground"),
     filename: "bootstrap.js",
   },
-  mode: "development",
```

Comment:
We should keep this as development, for production, you can just use `npm run build -- --production` in CI.